modules/ROOT/pages/custom-log-appender.adoc

```diff
@@ -44,6 +44,7 @@ You can use any Log4j appender. For information about configuring `log4j2.xml`,
 https://logging.apache.org/log4j/2.x/manual/configuration.html#ConfigurationSyntax[Log4j Configuration Syntax].
 
 To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings.
+Include the following Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` as shown in the example below along with your own logger settings.
```

Comment:
Can we change these lines like this:

To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings and include the Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile`.
In Anypoint Studio, the `log4j2.xml` file is located in the `src/main/resources` directory.